toolbox/src/sharedlibpp/CMakeLists.txt

```diff
@@ -43,5 +43,5 @@ include(InstallBasicPackageFiles)
 install_basic_package_files(sharedlibpp
     VERSION ${PROJECT_VERSION}
     COMPATIBILITY AnyNewerVersion
-    TARGETS sharedlibpp
+    EXPORT sharedlibppExport
```

Comment:
...but if you change `sharedlibpp` you should pass `FIRST_TARGET` here.

Reply:
Good catch, thanks!